reshard_checkpoints.py

```diff
@@ -0,0 +1,97 @@
+import os
+import shutil
+import torch
+import fire
+import json
+from pathlib import Path
+
+from fairscale.nn.model_parallel.utils import divide_and_check_no_remainder, split_tensor_along_last_dim
+
+from llama import ModelArgs, Transformer, Tokenizer
+from llama.xla_model_parallel import (
+    ParallelEmbedding,
+    RowParallelLinear,
+    ColumnParallelLinear,
+)
+
+@torch.no_grad()
+def reshard(original_mp, target_mp, ckpt_dir, output_dir, tokenizer_path):
```

Comment:
Can we have an example usage in the README?

Reply:
yes, added